### PR TITLE
docs(store-devtools): add docs for `connectOutsideZone` option

### LINKED ISSUE #4008: Docs: Add documentation for opting out of Zone.js connection with StoreDevtools
### Information

In https://github.com/ngrx/platform/issues/3839, we introduced a new option in StoreDevtools to connect to the Redux DevTools Extension outside the Zone to stop change detection from being triggered by the communication between the two.

```ts
provideStoreDevtools({
  connectOutsideZone: false,
})
```

This should be added to the examples on the overview page for Module and Standalone APIs.

https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/store-devtools/index.md

image

### Documentation page

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by va-stefanek**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

Closes #4008 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by va-stefanek** [comment]
@markostanimirovic I have added simple documentation based on the comments in code and issues in scope of which that config field has been added. If I miss something important in scope of the functionality it applies, please correct me.

**Review by markostanimirovic — CHANGES_REQUESTED** [review]

File: projects/ngrx.io/content/guide/store-devtools/index.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ bootstrapApplication(AppComponent, {
       autoPause: true, // Pauses recording actions and state changes when the extension window is not open
       trace: false, //  If set to true, will include stack trace for every dispatched action, so you can see it in trace tab jumping directly to that part of code
       traceLimit: 75, // maximum stack trace frames to be stored (in case trace option was provided as true)
+      connectOutsideZone: true // If set to true, the connection with dev tools extension will be estabilished in scope Angular zone
```
[the remark this answers is not in the document]
Comment (va-stefanek) [reply projects/ngrx.io/content/guide/store-devtools/index.md:58]: Wow, I totally confused this even though the name suggested the exact opposite 🤕

[the remark this answers is not in the document]
Comment (markostanimirovic) [reply projects/ngrx.io/content/guide/store-devtools/index.md]: Btw, you don't need to squash commits before push. It's easier for us to track requested changes when they are pushed as a separate commit.

**Review by timdeschryver — APPROVED** [review]
✨

**Pull request merged**